README.md

```diff
@@ -26,9 +26,18 @@ More information about building Idris from source has been detailed in the [Inst
 
 Idris has support for external code generators. Supplied with the distribution
 is a C code generator to compile executables, and a JavaScript code generator
-with support for node.js and browser JavaScript.
+with support for `node.js` and browser JavaScript.
 
-More information about [code generators can be found on the wiki](https://idris.readthedocs.io/en/latest/reference/codegen.html).
+More information about code generators can be found
+[here](https://idris.readthedocs.io/en/latest/reference/codegen.html).
+
+## Status
+
+This is Idris 1, implemented in Haskell. Idris 1 is not actively
+worked on anymore.
+
+[Idris 2](https://github.com/idris-lang/Idris2) is an early preview of
+the next generation of Idris, implemented in Idris 1.
```

Comment:
Idris 2 is implemented in Idris 2, it was bootstrapped from Idris2-boot, which is the one implemented in Idris 1

Reply:
Also, it's probably less unstable than an "early preview" now. But I don't know how Edwin would characterize it.